spec/features/idv/previous_address_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+feature 'IdV with previous address filled in' do
+  include IdvHelper
+
+  before do
+    @user = sign_in_and_2fa_user
+    visit verify_session_path
+    fill_out_idv_form_ok
+  end
+
+  context 'previous address has bad value' do
+    it 'fails resolution' do
+      fill_out_idv_previous_address_fail
+      click_idv_continue
+
+      expect(current_path).to eq verify_session_path
+    end
+  end
+
+  context 'neither address has bad value' do
+    it 'uses current address in final profile' do
+      fill_out_idv_previous_address_ok
+      click_idv_continue
+
+      fill_out_financial_form_ok
+      click_idv_continue
+
+      fill_out_phone_form_ok(@user.phone)
+      click_idv_continue
+
+      expect(current_path).to eq verify_review_path
+      expect(page).to have_content('123 Main St')
+      expect(page).to_not have_content('456 Other Ave')
+    end
+  end
+end
```

Comment:
Since these are slow specs, what I would do is have one big Idv flow spec that tests all the different scenarios in the same spec, like [this one](https://github.com/18F/identity-idp/blob/master/spec/features/idv/flow_spec.rb#L84). What we're mainly interested in is that the user can fill in the expected fields on this form, and that any error messages are displayed. We want to make sure that an error shows up when either the current or previous address is invalid.

I'm thinking something like this:
```
visit verify_session_path
fill_out_idv_form_ok
fill_out_idv_previous_address_fail
click_idv_continue

expect(current_path).to eq verify_session_path
expect(page).to have_content('Bad zipcode')

fill_out_idv_previous_address_ok
some_method_that_fills_out_current_address_with_bad_zipcode
click_idv_continue

expect(current_path).to eq verify_session_path
expect(page).to have_content('Bad zipcode')

fill_out_current_address_with_good_zipcode
click_idv_continue

fill_out_financial_form_ok
click_idv_continue

fill_out_phone_form_ok(@user.phone)
click_idv_continue

expect(current_path).to eq verify_review_path
expect(page).to have_content('123 Main St')
expect(page).to_not have_content('456 Other Ave')